multimodal/src/autogluon/multimodal/models/conv_lora/moe_conv.py

```diff
@@ -0,0 +1,238 @@
+from typing import Any, Callable, Dict, Iterator, List, Mapping, Optional, Set, Tuple, TypeVar, Union, overload
+
+import numpy as np
+import torch
+from torch import nn
+from torch.distributions.normal import Normal
+
+
+class MoEConv(nn.Module):
```

Comment:
Need a more accurate name? `MoEGate`? This class doesn't contain convolutions and is to determine gates.